scripts/perf/perf_log_check.py

```
@@ -328,10 +344,15 @@ def main() -> int:
     print(f"  rows={len(rows)} (analyzed {len(body)} after {args.warmup_rows} warmup)")
     print(f"  underruns={underruns}  produced_max={produced_max:.1f}ms  "
           f"catchup_bursts={catchup}  snap_forwards={snaps}")
-    print(f"  cost_p95={cost_p95:.2f}ms (emulation work)  produced_dropped={dropped}")
-    # Three states, not two: a pre-F16 log has no column, and reporting that as
-    # "VALID — window was on screen" would assert something never measured. The
-    # gate still passes it (see above), but it must not claim to have checked.
+    # v2.3.3 F17 — the NTSC frame period, the deadline `cost_p95` is racing.
+    ntsc_budget_ms = 16.639
+    util = 100.0 * cost_p95 / ntsc_budget_ms if cost_p95 == cost_p95 else float("nan")
```

Ruff (0.16.1)

[warning] Name compared with itself, consider replacing `cost_p95 == cost_p95`

(PLR0124)